portfolio.html

```diff
@@ -0,0 +1,236 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Raleway:wght@700&family=Roboto:wght@400;500;700;900&display=swap"
+    rel="stylesheet">
+  <link rel="stylesheet" href="https://yarnpkg.com/en/package/normalize.css">
+  <link rel="stylesheet" href="./css/styles.css">
+  </head>
+  <body>
+    <header class="header">
+      <div class="container-header">
+        <nav class="nav-header">
+          <a href="index.html" class="logo"><span class="logo-accent">Web</span>Studio</a>
+          <ul class="site-nav list">
+            <li class="item"><a href="./index.html" class="link">Студия</a></li>
+            <li class="item"><a href="./portfolio.html" class="link current" class="link">Портфолио</a></li>
+            <li class="item"><a href="#" class="link">Контакты</a></li>
+          </ul>
+          <ul class="site-cont list">
+            <li class="item">
+              <a href="mailto:info@example.com" class="link">
+                <svg class="icon mail">
+                  <use href="./images/symbol-defs.svg#icon-mail"></use>
+                </svg>info@example.com
+              </a>
+            </li>
+            <li class="item">
+              <a href="tel:+380991111111" class="link">
+                <svg class="icon phone">
+                  <use href="./images/symbol-defs.svg#icon-phone"></use>
+                </svg>+38 099 111 11 11
+              </a>
+            </li>
+          </ul>
+          </nav>
+          </div>
+          </header>
+    <main>
+      <section class="section">
```

Comment:
classname for the section has to be obvious. in case of naming. I see that here you have list of works , so section class="works" sounds more descriptive than section class="section"

Reply:
I called class=section-work